dallinger/prolific.py

```diff
@@ -529,6 +550,13 @@ def _req(self, method: str, endpoint: str, **kw) -> dict:
                         "external_study_url": "external-study-url",
                     }
 
+                elif re.match(r"/studies/[a-z0-9]+/screen-out-submissions/"):
+                    # method="POST", endpoint= "/studies/{study_id}/screen-out-submissions/", json=payload
+                    response = {
+                        "message": "The request to bulk screen out has been made successfully.",
+                        "payment_per_participant": {"amount": 15, "currency": "GBP"},
```

Comment:
`payment_per_participant` should be taken from the actual request, no?

Reply:
Done.

Reply:
@pmcharrison Maybe have a last look?